File: app/src/main/java/com/easym/vegie/model/shopbyquotation/Result.java
package com.easym.vegie.model.shopbyquotation;

import java.util.List;

public class Result {

    private String name;
    private String other_category_name;
    private List<Product> product = null;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<Product> getProduct() {
        return product;
    }

    public void setProduct(List<Product> product) {
        this.product = product;
    }

    public String getOther_category_name() {
        return other_category_name;
    }

    public void setOther_category_name(String other_category_name) {
        this.other_category_name = other_category_name;
    }
}

File: app/src/main/java/com/easym/vegie/model/mycartlist/Result.kt
package com.easym.vegie.model.mycartlist

class Result {

    private var name: String? = null
    private var product: List<Product?>? = null
    private var subtotal : Any?= null
    private var other_category_name : String?= null

    fun getName(): String? {
        return name
    }

    fun setName(name: String?) {
        this.name = name
    }

    fun getProduct(): List<Product?>? {
        return product
    }

    fun setProduct(product: List<Product?>?) {
        this.product = product
    }

    fun getSubtotal() : Any? {
        return subtotal
    }

    fun setSubtotal(subtotal: Any?) {
        this.subtotal = subtotal
    }


    fun getOtherCategoryName() : String? {
        return other_category_name
    }

    fun setOtherCategoryName(other_category_name: String?) {
        this.other_category_name = other_category_name
    }
}